pkg/storage/etcd3/store.go

```diff
@@ -57,6 +57,10 @@ type objState struct {
 	data []byte
 }
 
+func NewStore(c *clientv3.Client, codec runtime.Codec, prefix string) storage.Interface {
```

Comment:
> maybe just promote the existing newStore to be "New".

I will do this at the end because it will introduce noise from tests. Let's use New() right now.